### PR TITLE
Fixed building for device, documented workaround for building for simulator

### PR DESCRIPTION
**Pull request by sleeke**

### CODE REVIEW

File: SampleApp/package.json [hunk 1]
```diff
@@ -30,6 +32,7 @@
     "cordova-plugin-android-permissions": "^1.1.5",
     "cordova-plugin-androidx-adapter": "^1.1.3",
     "cordova-plugin-badge": "^0.8.8",
-    "cordova-plugin-device": "^2.1.0"
+    "cordova-plugin-device": "^2.1.0",
+    "Verint-XM": "file:../../verint-xm-cordova-plugin"
```
Comment (wayneburkett) [inline SampleApp/package.json:36]: Was this meant to be committed?

File: SampleApp/build.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (wayneburkett) [inline SampleApp/build.json:1]: Is this available for client use, too?

Comment (sleeke) [reply SampleApp/build.json:1]: No, but it serves as an example of how to have it set up, and the changes in the readme should cover the changes they'd need to make.

File: SampleApp/package.json (hunk 1 above)
Comment (sleeke) [reply SampleApp/package.json:36]: I hadn't thought it mattered too much, but now I'm remembering our chat. I'll remove it.

**Pull request merged**